esiclient/tests/unit/v1/test_node_network.py

```diff
@@ -469,6 +473,24 @@ def test_take_action_no_port_or_network_or_trunk_exception(self):
             'ERROR: You must specify either network, port, or trunk',
             self.cmd.take_action, parsed_args)
 
+    def test_take_action_invalid_network_exception(self, mock_gfnifp):
```

Comment:
You're missing the decorator here - actually, you probably don't need mock_gfnifp if you're not using it.

Reply:
yes I removed it, I don't think I need to use it here. It was here by mistake because I was trying something when it didn't work.